### PR TITLE
WikibaseRepo: add business context diagram

### PR DESCRIPTION
**Pull request by jakobw**

### CODE REVIEW

**Review by addshore — CHANGES_REQUESTED** [review]
This diagram continues to have a thicker arrow for entity data going to the client system.
Perhaps it should also have a thicker arrow for the query service?
Or we should get rid of the difference in arrow thickness here and in client.

Im curious to figure out what we are actually defining as "entity data" as we keep throwing that term around but probably didn't define it?
It seems quite broad, but maybe that is okay?

IMO it's probably good enough to merge and then we can iterate on it going forward.

**Comment by jakobw** [comment]
> This diagram continues to have a thicker arrow for entity data going to the client system.
Perhaps it should also have a thicker arrow for the query service?
Or we should get rid of the difference in arrow thickness here and in client.

Yeah, let's just use the same arrows everywhere. Fixed :)

> Im curious to figure out what we are actually defining as "entity data" as we keep throwing that term around but probably didn't define it?
It seems quite broad, but maybe that is okay?

It's probably ok to be broad in this diagram as long as we remember to break it down in the lower levels. What do you think?

File: systems/WikibaseRepo/03-Context_and_Scope.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 ### Business Context
 
+![Wikibase Repo business context diagram](./diagrams/03-business-context.drawio.svg)
+
```
Comment (Silvan-WMDE) [inline systems/WikibaseRepo/03-Context_and_Scope.md:6]: are we still missing the Neighbour/Description table here?

File: systems/WikibaseRepo/diagrams/03-business-context.drawio.svg [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        </g>
+        <path d="M 38.12 220 L 220 220" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/>
+        <path d="M 31.12 220 L 38.12 217.67 L 38.12 222.33 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 211px; margin-left: 131px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 11px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">
+                                read Entity data
```
Comment (Silvan-WMDE) [inline systems/WikibaseRepo/diagrams/03-business-context.drawio.svg:40]: minor nitpick: the User read**s** and edit**s** entity data, as opposed to the Tools and Bots, which read and edit

**Review by Silvan-WMDE — CHANGES_REQUESTED** [review]
The diagram looks good to me, but description should be added

File: systems/WikibaseRepo/03-Context_and_Scope.md (hunk 1 above)
Comment (jakobw) [reply systems/WikibaseRepo/03-Context_and_Scope.md:6]: done!

File: systems/WikibaseRepo/diagrams/03-business-context.drawio.svg (hunk 2 above)
Comment (jakobw) [reply systems/WikibaseRepo/diagrams/03-business-context.drawio.svg]: done!

**Review by Silvan-WMDE — APPROVED** [review]

**Pull request merged**